c++/src/kj/array.h

```diff
@@ -189,6 +189,14 @@ class Array {
     KJ_IREQUIRE(start <= end && end <= size_, "Out-of-bounds Array::slice().");
     return ArrayPtr<const T>(ptr + start, end - start);
   }
+  inline ArrayPtr<T> slice(size_t start) {
```

Comment:
you'll need to add KJ_LIFETIMEBOUND like above.

Reply:
Fixed in https://github.com/capnproto/capnproto/compare/d4db83d435411a7e34262e1e21c180f8b386e969..e8bb906f6c2b58491f62d3ef439141cdb91a2e3d.